src/index.html

```diff
@@ -1,22 +1,455 @@
+          href="#"
+          class="explore-online__link"
+        >
+          <img
+            class="explore-online__photo"
+            src="./images/image10-square.png"
+            alt="image"
+          />
+        </a>
+        <div class="explore-online__text">
+          <h3 class="explore-online__text-title">Virtual group tours</h3>
+          <p class="explore-online__text-description">
+            Request a guided tour, reserve admission and lead your own small
+            group, or request a live virtual tour led by a Museum guide.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section
+      class="contacts"
+      id="contacts"
+    >
+      <h2 class="contacts-title">Contact us</h2>
+
+      <div class="contacts__container__wrapper">
+        <div class="contacts__container">
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Fifth Avenue</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a
+                href="tel:+1 212-535--7710"
+                class="contacts__title--content"
+              >
+                +1 212-535--7710
+              </a>
+              <p class="contacts__title--title">Address</p>
+              <a
+                href="https://maps.app.goo.gl/zW1hK1uyAj9jKZiR9"
+                target="_blank"
+              >
+                <p>
+                  1000 Fifth Avenue
+                  <br />
+                  New York, NY 10028
+                </p>
+              </a>
+            </div>
+          </div>
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Cloisters</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a
+                href="tel:+1 212-923-3700"
+                class="contacts__title--content"
+              >
+                +1 212-923-3700
+              </a>
+              <p class="contacts__title--title">Address</p>
+              <a
+                href="https://maps.app.goo.gl/Dd9D2kouF1f8HZ5r9"
+                target="_blank"
+              >
+                <p>
+                  99 Margaret Corbin Drive
+                  <br />
+                  Fort Tryon Park
+                  <br />
+                  New York, NY 10040
+                </p>
+              </a>
+            </div>
+          </div>
+        </div>
+
+        <form>
+          <input
+            type="text"
+            placeholder="Your Name Complete"
+            required
+          />
+          <input
+            type="email"
+            placeholder="email@gmail.com"
+            required
+          />
+          <textarea
+            type="review"
+            placeholder="Type your message here."
+            required
+          ></textarea>
+          <button
+            class="button"
+            type="reset"
```

Comment:
Your button should submit, use `<form onsubmit="this.reset(); return false;">` to clear it upon submission (this is done to test input type validation and required flag, if you set your button to reset it will skip all validations and clear the inputs) [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=The%20page%20shouldn%27t%20be%20reloaded%20on%20form%20submit%2C%20but%20the%20fields%20should%20be%20cleared.%20Use%3A)

Reply:
In the last review, they told me to use the button reset (?), but okay :D